wikilinksbot.py

```diff
@@ -7,7 +7,7 @@
 gc.enable()
 updater = Updater(bot_config.token, use_context=True)
 # The main regex we use to find linkable terms in messages
-regex = re.compile(r"(\[\[.+?[\||\]]|(?<!\w)(?<!\w[=/])(?<!wiki/)(?<!Property:)(?<!Lexeme:)(?<!EntitySchema:)(?<!title=)(L[1-9]\d*(-[SF]\d+)|[QPLET][1-9]\d*(#P\d+)?))")
+regex = re.compile(r"(\[\[.+?[\||\]]|(?<!\w)(?<!\w[=/])(?<!^/)(?<!Property:)(?<!Lexeme:)(?<!EntitySchema:)(?<!Item:)(?<!title=)(L[1-9]\d*(-[SF]\d+)|[QPLETM][1-9]\d*(#P[1-9]\d*)?))")
```

Comment:
The reason "wiki/" was specified as opposed to just "/" is that with the latter, cases like someone saying "You should use Q6581097/Q6581072" would not link the second item.

Reply:
I removed the `?<!wiki/` as it is already fulfilled by `?<\w[=/])`. Your original regex did not even work with Q6581097/Q6581072.

Reply:
Oh, I was sure `\w` was the same as `[A-Za-z]`, but it also includes `[0-9_]`. TIL! Yes, 2/Q5 and t/Q5 are supposed to work, but I obviously hadn't tested that properly! :-) `?<!^/` makes sense.